### PR TITLE
Raven db 12955

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: src/Raven.Server/Web/RequestHandler.cs [hunk 1]
```diff
@@ -573,18 +574,57 @@ private static void ThrowInvalidAuthStatus(RavenServer.AuthenticationStatus? sta
...
 
+        private static bool IsOriginAllowed(string origin, ServerStore serverStore)
+        {
+            if (serverStore.Server.Certificate.Certificate == null)
+            {
+                // running in unsafe mode - since server can be access via multiple urls/aliases accept them 
+                return true;
+            }
+            
+            var topology = serverStore.GetClusterTopology();
```
Comment (ayende) [inline src/Raven.Server/Web/RequestHandler.cs:604]: Do we need to worry about case sensitivity here?

Comment (ml054) [reply src/Raven.Server/Web/RequestHandler.cs:604]: Good question, I will check that using empirical mode. :)

**Comment by ml054** [comment]
I've found one bug. Closing for now.

File: src/Raven.Server/Web/RequestHandler.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Server/Web/RequestHandler.cs:604]: It needs to be case insensitive. Thanks. I will fix that.

**Pull request closed without merging**